src/server/api/controllers/modules.controller.crop-stages-endpoint.js

```diff
@@ -0,0 +1,24 @@
+const knex = require('../../config/db');
+
+const getCropStages = async () => {
+  try {
+    const startDate = await knex('batches')
+      .select('seeding_date')
+      .where({ id: '31' });
+    const stages = await knex('crop_stages').select(
+      'crop_stages.name',
+      'crop_stages.duration',
+    );
```

Comment:
Is this correct? Do you need to query all entries?

Reply:
stages I guess a gonna be same every time  ... but yes we need to get all the stages the crop is gonna pass through ... because we need to calculate total days of crop cycle ..and also the stage name to be shown under progress bar .. 
any suggestions ?